apps/console/test-configs/mocks/mock.deployment.config.json

```diff
@@ -0,0 +1,187 @@
+{
```

Comment:
Shall we name the folder containing mocks as `__mocks__`?

Reply:
Resolved with commit 35d9fc9